alecs-theme/src/WEB-INF/src/resources-importer/journal/structures/Service.json

```diff
@@ -0,0 +1,191 @@
+{
+    "availableLanguageIds": [
+        "en_US"
+    ],
+    "defaultLanguageId": "en_US",
+    "fields": [
+        {
+            "label": {
+                "en_US": "Background Color"
+            },
+            "predefinedValue": {
+                "en_US": ""
+            },
+            "style": {
+                "en_US": ""
+            },
+            "tip": {
+                "en_US": ""
+            },
+            "dataType": "color",
+            "fieldNamespace": "ddm",
+            "indexType": "keyword",
+            "localizable": true,
+            "name": "BackgroundColor",
+            "readOnly": false,
+            "repeatable": false,
+            "required": true,
+            "showLabel": true,
+            "type": "ddm-color"
+        },
+        {
+            "label": {
+                "en_US": "Fade In Delay"
+            },
+            "predefinedValue": {
+                "en_US": "1000"
+            },
+            "style": {
+                "en_US": ""
+            },
+            "tip": {
+                "en_US": ""
+            },
+            "dataType": "integer",
+            "fieldNamespace": "ddm",
+            "indexType": "keyword",
+            "localizable": true,
+            "name": "FadeInDelay",
+            "readOnly": false,
+            "repeatable": false,
+            "required": true,
+            "showLabel": true,
+            "type": "ddm-integer"
+        },
```

Comment:
Should we use a select list to predefine and limit the available options for the end user?  Add a predefined value, tip or some description so the end user knows what to do.